### PR TITLE
components: 02-elements: cookie-consent: check if there are cookies

### PR DESCRIPTION
**Pull request by BibianaC**
relates to #63

### CODE REVIEW

**Comment by BibianaC** [comment]
@michaelwood not sure how to test if this works.

**Review by michaelwood — CHANGES_REQUESTED** [review]
It may be possible to test some of this by pasting it to an application. If we need to test fully we could use the current 360dev server with a threesixtygiving domain name.

File: src/components/02-elements/cookie-consent/cookie-consent.njk [hunk 1]
```diff
@@ -20,6 +20,39 @@
...
+          document.cookie = "noEnhancedAnalytics=1; domain=threesixtygiving.org"
+        }
+
+        function getCookie() {
+          var decodedCookie = decodeURIComponent(document.cookie);
+          var cookieArray = decodedCookie.split(';');
+
+          var cookieValue = [];
+
+          for(var i = 0; i < cookieArray.length; i++){
```
[the remark this answers is not in the document]
Comment (BibianaC) [reply src/components/02-elements/cookie-consent/cookie-consent.njk]: Thanks, @michaelwood. I have simplified it.

File: src/components/02-elements/cookie-consent/cookie-consent.njk [hunk 2]
```diff
@@ -20,6 +20,39 @@
...
+              var firstEq = cookieSeg.indexOf("=");
+              var value = cookieSeg.substr(firstEq+1);
+              cookieValue.push(value);
+           }
+           return cookieValue;
+        }
+
+        function checkCookie() {
+          var cookieValue = getCookie();
+          ["rememberCookieConsentGiven", "optUserOut", "noEnhancedAnalytics"].forEach(cookieExist);
```
[the remark this answers is not in the document]
Comment (BibianaC) [reply src/components/02-elements/cookie-consent/cookie-consent.njk]: Thanks

**Pull request merged**